modules/islandora_scholar_embargo/islandora_scholar_embargo.module

```diff
@@ -474,7 +483,10 @@ function islandora_scholar_embargo_set_embargo($param, $dsid = NULL, $end = 'ind
       }
       foreach ($users as $user) {
         $xacml->viewingRule->addUser($user->name);
-        $xacml_changed = TRUE;
+      }
+      foreach ($roles as $role) {
+        $xacml->viewingRule->addRole($role);
+        $xacml->managementRule->addRole($role);
```

Comment:
Is management really necessary? How could others manage something they can't see?